tests/CMakeLists.txt

```diff
@@ -201,13 +214,20 @@ add_test(NAME test_build_all_off
 add_test(NAME test_build_all_on
         COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options ${PROJECT_SOURCE_DIR} test_build_all_on "-DBUILD_WITH_CODE_COVERAGE=ON -DBUILD_GENIVI=ON -DBUILD_OSTREE=ON")
 
+add_test(NAME test_build_ci_tests_on
+        COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options ${PROJECT_SOURCE_DIR} test_build_ci_tests_on "-DBUILD_WITH_CODE_COVERAGE=OFF -DBUILD_GENIVI=OFF -DBUILD_OSTREE=ON -DPROVISION_FILE=${PROJECT_BINARY_DIR}/test_data/credentials.zip")
+
```

Comment:
I wouldn't worry about this test case--if there _is_ a provisioning file, then we can use it, and if the _isn't_ a provisioning file we can't do this test any way :)